modules/monitoring/sns.tf

```diff
@@ -1,8 +1,8 @@
 # SNS Topic
-
```

Comment:
Why did you remove the empty line?

Reply:
Saw an empty line and removed :D Brought it back